mappings/net/minecraft/predicate/PlayerPredicate.mapping

```diff
@@ -0,0 +1,45 @@
+CLASS net/minecraft/class_4553 net/minecraft/predicate/PlayerPredicate
+	CLASS class_4554 AdvancementCriteriaPredicate
+		FIELD field_20728 criteria Lit/unimi/dsi/fastutil/objects/Object2BooleanMap;
+		METHOD <init> (Lit/unimi/dsi/fastutil/objects/Object2BooleanMap;)V
+			ARG 1 criteria
+		METHOD test (Ljava/lang/Object;)Z
+			ARG 1 progress
+	CLASS class_4555 CompletedAdvancementPredicate
+		FIELD field_20729 done Z
+		METHOD <init> (Z)V
+			ARG 1 done
+		METHOD test (Ljava/lang/Object;)Z
+			ARG 1 progress
+	CLASS class_4556 AdvancementPredicate
+		METHOD method_22506 toJson ()Lcom/google/gson/JsonElement;
+	CLASS class_4557 Builder
+		FIELD field_20730 experienceLevel Lnet/minecraft/class_2096$class_2100;
+		FIELD field_20731 gamemode Lnet/minecraft/class_1934;
+		FIELD field_20732 stats Ljava/util/Map;
+		FIELD field_20733 recipes Lit/unimi/dsi/fastutil/objects/Object2BooleanMap;
+		FIELD field_20734 advancements Ljava/util/Map;
+		METHOD method_22507 build ()Lnet/minecraft/class_4553;
+	FIELD field_20722 ANY Lnet/minecraft/class_4553;
+	FIELD field_20723 experienceLevel Lnet/minecraft/class_2096$class_2100;
+	FIELD field_20724 gamemode Lnet/minecraft/class_1934;
```

Comment:
`gameMode` (this capitalization is used in the class name and fields in other classes)

Reply:
https://minecraft.gamepedia.com/Java_Edition_19w38a#General
It's called `gamemode` in JSON key, so `gamemode` is fine for field name

Reply:
Imo should have a standard still -- maybe can rename other fields to `gamemode` like this, or alternatively change this name?